packages/devtools_app/lib/src/screens/memory/shared/primitives/memory_timeline.dart

```diff
@@ -117,16 +84,6 @@ class MemoryTimeline {
     _eventSamples.add(event);
   }
 
-  void addSnapshotEvent({bool auto = false}) {
```

Comment:
@polina-c this also seems like a bug that we have stopped adding an event to the timeline when we take a snapshot. Maybe the action item here is to use this code.

Reply:
Same as https://github.com/flutter/devtools/pull/5949#discussion_r1342165085

Reply:
@polina-c please advise

Reply:
This one as well